tests/admin_inlines/admin.py

```diff
@@ -23,6 +23,11 @@ class NonAutoPKBookTabularInline(admin.TabularInline):
     classes = ('collapse',)
 
 
+class NonAutoPKBookDescendantTabularInline(admin.TabularInline):
+    model = NonAutoPKBookDescendant
+    classes = ('collapse',)
```

Comment:
I can take it out but would still need to adapt that test because it also checks for "Hide" which changes since this would now be open by default.